testsuite/shell.go

```diff
@@ -259,8 +259,14 @@ func ExecuteCommandWithRetry(retryCount int, retryTime string, command string, e
 	for i := 0; i < retryCount; i++ {
 		err := ExecuteCommand(command)
 		exitCode, stdout := shell.excbuf.String(), shell.outbuf.String()
-		if err == nil && exitCode == "0" && strings.Contains(stdout, expected) {
-			return nil
+		if strings.Contains(containsOrNot, "not") {
```

Comment:
I do not think this is right. It would match anything that contains `not`, like `note`, `annotate`, `notifies`, etc

Reply:
That should not happen because this function won't be executed if `containsOrNot` is not matched to one of the four options: `should contain`, `contains`, `should not contain`, `does not contain`. So we should be good.